src/app/(events)/events/[id]/page.tsx

```
@@ -8,7 +8,7 @@ import {
   CardTitle,
 } from '@/components/shadcn-ui/card';
 import React, { Suspense } from 'react';
-import { Calendar, Map, SquareArrowOutUpRight } from 'lucide-react';
+import { Calendar, Map, SquareArrowOutUpRight, User } from 'lucide-react';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)